compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.cpp

```diff
@@ -568,4 +569,13 @@ void eraseCompilationInfo(Operation *op) {
   op->removeAttr(kCompilationInfoAttrName);
 }
 
+//===----------------------------------------------------------------------===//
+// Helpers for setting `iree_codegen.root_op` attribute on root operations for
```

Comment:
There is no `iree_codegen.root_op` attribute. What you're doing is adding a unit attribute to the op. Perhaps rephrase it like "Helpers for setting attributes for tuner".